### PR TITLE
Pre-download newsletter experiment on /firefox (Fixes #5944)

### LINKED ISSUE #5944: Experiment - ask for email prior to download
[manapage w. test write up](https://mana.mozilla.org/wiki/display/FIREFOX/Ask+for+email+prior+to+download+test)

Success Criteria

- [ ] non-Firefox en-US visitors on Windows only
- [ ] Implemented on www.mozilla.org/firefox/

Prototype:
https://invis.io/SDNAAE3ZCK4

Sketch File:
https://share.goabstract.com/3473d71b-737b-480a-8dd0-658d84692668

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds experiment to show a newsletter signup modal when clicking "Download Firefox" on the `/firefox` page.

**do not merge** until we get both analytics and the new email privacy text in place.

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/5944

## Testing
Demo:
  - https://bedrock-demo-agibson.oregon-b.moz.works/en-US/firefox/

Direct URL variants:
  - https://bedrock-demo-agibson.oregon-b.moz.works/en-US/firefox/?v=a (control)
  - https://bedrock-demo-agibson.oregon-b.moz.works/en-US/firefox/?v=b (newsletter + headline 1)
  - https://bedrock-demo-agibson.oregon-b.moz.works/en-US/firefox/?v=c (newsletter + headline 2)

### CODE REVIEW

File: jenkins/branches/demo/agibson.env [hunk 1]
```diff
@@ -0,0 +1 @@
+STUB_ATTRIBUTION_HMAC_KEY='foo'
```
Comment (alexgibson) [inline jenkins/branches/demo/agibson.env]: Self-reminder to remove this before merging.

**Comment by stephaniehobson** [comment]
Two UX concerns I don't think should be a blocker to this experiment but should be addressed if it is a success and we want to make it permanent:

1. We should not ask for email address again after a successful submission
1. The newsletter success message goes away too quickly to read but too slow to not make me want to read it.

**Comment by alexgibson** [comment]
Ok, this finally has analytics wrapped up and is ready for code review / merge. The ask is for this to go live on Thursday this week.

File: media/css/firefox/new/scene2.scss [hunk 2]
```diff
@@ -2,45 +2,15 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-@import '../../pebbles/includes/lib';
-@import '../../hubs/sections';
+$font-path: '/media/fonts';
```
Comment (alexgibson) [inline media/css/firefox/new/scene2.scss]: This diff here is the result of renaming the scss file. There are no code changes here.

**Comment by stephaniehobson** [comment]
I don't know if this would be difficult so I won't block on it but on the email sign up failure screen it would be nice to add an explicit link to continue the download next to the error message:

screen shot 2018-09-19 at 3 38 59 pm

The escape hatch is "below the fold" 🦆

**Comment by alexgibson** [comment]
> I don't know if this would be difficult so I won't block on it but on the email sign up failure screen it would be nice to add an explicit link to continue the download next to the error message

I think this would be a good feature to add if this experiment ever becomes a reality, but for the purposes of this test it may be worth evaluating just how impactful this whole flow already is on user experience. There are so many barriers to entry & levels of frustration here, that I don't think we can patch all the holes without redesigning the UX entirely 😞

**Comment by alexgibson** [comment]
@stephaniehobson thanks for the review, updated

**Pull request merged**